src/connectors/fname/mod.rs

```diff
@@ -161,6 +167,7 @@ impl Fetcher {
                     fid: t.to,
                     r#type: UserNameType::UsernameTypeFname as i32,
                 };
+                self.count("num_fname_transfers_ingested", 1);
```

Comment:
We keep metric names concise by convention. So something like `fnames.transfers`